### PR TITLE
COS-3063: Add kubesan host dependencies

### PR DESCRIPTION
**Pull request by copejon**
See:
- [OCPEDGE-1418](https://issues.redhat.com//browse/OCPEDGE-1418)
- [COS-3062](https://issues.redhat.com/browse/COS-3062)

### CODE REVIEW

File: extensions-ocp-rhel-9.6.yaml [hunk 1]
```diff
@@ -88,3 +88,9 @@ extensions:
   sysstat:
     packages:
       - sysstat
+  sanlock:
+    packages:
+      - sanlock
+  lvm2-lockd:
+    packages:
+      - lvm2-lockd
```
[the remark this answers is not in the document]
Comment (brandisher) [reply extensions-ocp-rhel-9.6.yaml:96]: It looks like these should be under one extension from what I can infer from the docs; you need lvm2-lockd to enable LVM locking and then a lock manager, sanlockd in this case.

Are there recommendations on how to name the extension? Should we just be calling it `ksan` (the operator name) or something else?

[the remark this answers is not in the document]
Comment (mike-nguyen) [reply extensions-ocp-rhel-9.6.yaml:96]: We don't have any official naming conventions. I'd prefer `kubesan` over `ksan` because it's more explicit.

**Comment by jeff-roche** [comment]
This  PR will close in favor of https://github.com/openshift/os/pull/1701 once @copejon is back from PTO

**Pull request closed without merging**